components/resources/brave_components_strings.grd

```diff
@@ -1032,6 +1032,7 @@
       <part file="decentralized_dns_strings.grdp" />
       <part file="domain_block_strings.grdp" />
       <part file="ipfs_strings.grdp" />
+      <part file="trezor_bridge_strings.grdp" />
```

Comment:
nit: I think we should not keep adding to this file, but make a unique one for the trezor-bridge component. Plus, this needs a conditional for if wallet is enabled.